spec/factories/lti_line_items.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :lti_line_item do
```

Comment:
I believe this should have an association with an `assessment` as well (which by default you could have be an `:assignment`?)